Source/WebCore/contentextensions/ContentExtensionActions.cpp

```diff
@@ -788,6 +789,8 @@ void RedirectAction::URLTransformAction::QueryTransform::applyToURL(URL& url) co
     for (auto& key : removeParams)
         keysToRemove.add(key);
     form.removeAllMatching([&] (auto& keyValue) {
+        if (keysToRemove.contains(keyValue.key))
```

Comment:
We should avoid calling `contains` twice here — either using a temporary variable to store the result or doing a `return true;` / `return false;`